HomeWork3/Nataliia_Kuchynska_HW3.cpp

```diff
@@ -0,0 +1,102 @@
+#include "Header.hpp"
+#include <iostream>
+#include <random>
+
+const rsize_t ksize = 100;
```

Comment:
Any variable that is used to represent the size of an object, including integer values used as sizes, indices, loop counters, and lengths, should be declared rsize_t, if available. Otherwise, it should be declared size_t.
https://wiki.sei.cmu.edu/confluence/display/c/INT01-C.+Use+rsize_t+or+size_t+for+all+integer+values+representing+the+size+of+an+object
П.С. или так нельзя?

Reply:
это наверное С шное?

Reply:
It might be a good idea to include the fact that rsize_t is defined in the C specification, but not in the C++ specification.